build.gradle

```diff
@@ -10,14 +10,30 @@ version '1.0-SNAPSHOT'
 repositories {
     mavenCentral()
 }
+gatling {
+    simulations = {
+        include "simulations/publication/*" // <1>
```

Comment:
I think we should change this to simulations/** to cover all the simulations folders

Reply:
Fixed :)